modelina-website/src/components/playground/Playground.tsx

```diff
@@ -223,10 +224,6 @@ class Playground extends React.Component<
       this.config.csharpIncludeJson =
         query.csharpIncludeJson === 'true';
     }
-    if (query.csharpIncludeNewtonsoft !== undefined) {
-      this.config.csharpIncludeNewtonsoft =
-        query.csharpIncludeNewtonsoft === 'true';
-    }
```

Comment:
This should not be removed 🙂 Think it was removed when you solved conflicts.

Reply:
I have resolved it @jonaslagoni! Thank you.